fiftyone/operators/types.py

```diff
@@ -971,8 +971,34 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
 
+class UploadedFile(dict):
```

Comment:
Can we add mapping similar to https://github.com/voxel51/fiftyone/pull/3586/files#diff-136b627e19a5efadd63b2ecebf3d40f59685b433ac4ab576c052c913531951a0R46

Reply:
This isn't a `BaseType` - but I could convert it to one. The auto mapping does infer that its a `types.Object` or `types.UploadedFile` though doesn't it? When the actual support is `string | types.Object | types.UploadedFile`...?